testnet/stacks-node/src/config.rs

```diff
@@ -668,6 +668,10 @@ impl Config {
         format!("{}/peer_db.sqlite", self.node.working_dir)
     }
 
+    pub fn get_atlas_db_path(&self) -> String {
+        format!("{}/atlas_db.sqlite", self.node.working_dir)
```

Comment:
Can we put the atlas DB within the chainstate directory?  Different chains will have different atlas databases.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/2b5095573b9daffe429868970089cd1a10329d2f